### ISSUE TITLE
Connector CPE - Use Title as name instead of the plain product name extract

### ISSUE THREAD

**Issue by iFrozenPhoenix** [OpenCTI-Platform/connectors#1931]
## Description
Currently the obserables name is extracted from the cpe string. In my opinion it should be the title (A readable name that also includes the version no).

## Environment

1. OS (where OpenCTI server runs): debian k8s
2. OpenCTI version: 6.0.5
3. OpenCTI client: connector
4. Other environment details:

## Reproducible Steps
N.A.

## Expected Output
An Obserable with a readable title

## Actual Output
An observable with a product name string (machine readable format; snake case)

## Additional information
I have a fix for this where the title is used as the name if an english title is available, else fallback to current behaviour.
Would make a PR if ok

**Comment by nino-filigran** [comment]
@iFrozenPhoenix just to make it clear: does the title always refer to the product name? It seems that in the XML specs, the title is optional (unless I'm referring to the wrong field "The optional title element is used to provide a human-readable title for the platform").
If empty, wouldn't it make sense to fallback to default behavior? Apologies if my comment does not make sense.

**Comment by iFrozenPhoenix** [comment]
@nino-filigran I have implemented it in that way that if an english title is available it uses the english title, if no english title is available it already falls back to the current behavior.
While it's correct that it is optional it is so far used in all of the cpes by nist.

**Comment by iFrozenPhoenix** [comment]
@nino-filigran could you please take a look on the PR? https://github.com/OpenCTI-Platform/connectors/pull/1933. If everything is ok I would provide the Bugfix for https://github.com/OpenCTI-Platform/connectors/issues/1927 after the merge.

**Comment by helene-nguyen** [comment]
Yes @nino-filigran, @iFrozenPhoenix, we will check it when we have time and give you an update on that :)

**Comment by d1zanv** [comment]
> ## Description
> 
> Currently the obserables name is extracted from the cpe string. In my opinion it should be the title (A readable name that also includes the version no).
> ## Environment
> 
>     1. OS (where OpenCTI server runs): debian k8s
> 
>     2. OpenCTI version: 6.0.5
> 
>     3. OpenCTI client: connector
> 
>     4. Other environment details:
> 
> 
> ## Reproducible Steps
> 
> N.A.
> ## Expected Output
> 
> An Obserable with a readable title
> ## Actual Output
> 
> An observable with a product name string (machine readable format; snake case)
> ## Additional information
> 
> I have a fix for this where the title is used as the name if an english title is available, else fallback to current behaviour. Would make a PR if ok

I don't know why I didn't take the title from the beginning. Thanks for the fix @iFrozenPhoenix !
